test/widget_tests/splash_screen_test.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:trovami/Strings.dart';
+import 'package:trovami/helpers/RoutesHelper.dart';
+import 'package:trovami/ui/splash/splash_screen.dart';
+
+void main() {
+  testWidgets('Shows splash screen for a few seconds and Navigates to Log in page', (WidgetTester tester) async {
```

Comment:
This test doesn't actually "Show the splash screen for a few seconds" ?
It displays the splash screen and waits for however long the splash screen is coded to stay up?
Just making sure I understand what pumpAndSettle does

Reply:
Lol, yeah I realized that too. I am kinda new to write tests too but just starting to know a lot more. I can probably test if the splashscreen widget was mounted that would test the appearance of splash screen I suppose.